enhancements/split_helm_ansible_into_their_repos.md

```diff
@@ -0,0 +1,283 @@
+| * hack/generate/samples/projects/*_ansible_sample.go 	|  hack/generate/samples/projects/*_ansible_sample.go |  It is not currently implemented. See the section `Operator Types Samples` on this document |
+
+### Usage of Helm/Ansible plugins in SDK 
+
+Helm and Ansible plugins will be imported and added to `operator-sdk`'s CLI much like they [currently are](https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/cli/cli.go#L62-L63), except from their respective repositories.
+
+However, note that in the [operator-sdk/internal/plugins](https://github.com/operator-framework/operator-sdk/tree/master/internal/plugins/) its implementation would be very similar to the [Go plugin](https://github.com/operator-framework/operator-sdk/tree/master/internal/plugins/golang/v2). SDK will consume the `helm-operator` and `ansible-operator` to build the projects much like its consume Kubebuilder to build the Go projects.   
+
+### CLI usage on Operator Types project 
+
+The binaries would also need to provide the CLI features to allow build the projects and test them without SDK CLI tool. However, this implementation came from Kubebuilder and its implementation would like it is done in [operator-sdk/internal/cmd/operator-sdk/cli/cli.go](https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/cli/cli.go). 
+
+### Update website SDK docs
+
+ In this way, if we can accomplish all goes described here before being able to create the Proposal to describe and specify the steps required to achieve the goal of the Operator Types projects have their own docs subdomains `ansible.operatorframework.io` and `helm.operatorframework.io` or if we decide not move forward with it as well we can define that the website docs should be updated when a new version of these repos are bumped on the CLI. 
+
+Note that, the above scenarios requires small changes in the docs usually. Also, changes in the tutorials will be more common when the upstream (Kubebuilder) implementation change and this case, also will be required update the website docs when a new Kubebuilder version be bumped. So, it shows achievable. 
+  
+### Operator Types Samples 
+
+See the proposal [Automatic Samples Generation using Go language](https://github.com/operator-framework/enhancements/pull/47/files). The implementation to generate the samples (SampleContext) as it helpers could be on upstream. 
+ 
+Then, each Operator Type should be able to build its own samples in the `testdata/` directory.
+
+### Scope of Tests per project
+
+- Operator Types (`helm-operator` and `ansible-operator`) should contains tests that will cover its libs and the project scaffold by them.  
+
+- SDK CLI tool (`operator-sdk`) should contain the test required to cover the CLI commands and e2e tests which will cover the specific customizations made on top of helm/ansible plugins for example olm integrations. 
+
```

Comment:
HI @estroz, 

I tried to put here what in my mind should be covered on the tests per project.